source/renderer/app/api/network/types.js

```diff
@@ -54,7 +54,7 @@ export type GetNetworkInfoResponse = {
   localTip: TipInfo,
   networkTip: ?TipInfo,
   nextEpoch: ?NextEpoch,
-  futureEpoch: ?FutureEpoch,
+  futureEpoch?: ?FutureEpoch,
```

Comment:
@yakovkaravelov shouldn't we remove `futureEpoch` from this type declaration?

Reply:
@yakovkaravelov fixed in https://github.com/input-output-hk/daedalus/pull/2165/commits/5f8b08992a9e2ccc49eb834beb46b2149b16c8ce